tap_impact/sync.py

```diff
@@ -1,13 +1,16 @@
-from datetime import timedelta
+from datetime import timedelta, datetime
 import json
 import singer
 from singer import metrics, metadata, Transformer, utils
-from singer.utils import strptime_to_utc, strftime
+from singer.utils import strptime_to_utc
 from tap_impact.transform import transform_json
 from tap_impact.streams import STREAMS
 
 LOGGER = singer.get_logger()
 BASE_URL = 'https://api.impact.com'
+# Window size implemented for actions and actionUpdates streams
+# as startDate and EndDate can just be 45 days apart
+DEFAULT_WINDOW_SIZE = 45
```

Comment:
any specific reason to keep it 45 days? rather than a standard 1 month or 30 days date window?

Reply:
This is the PR referenced in description which tells about this in detail: https://github.com/singer-io/tap-impact/pull/35